src/registry/IRegistry.ts

```diff
@@ -10,10 +10,22 @@ export interface ChainFiles {
 export interface RegistryContent {
   chains: ChainMap<ChainFiles>;
   deployments: {
-    // TODO define deployment artifact shape here
+    // Map of warp route ID to config URI
+    warpRoutes: Record<WarpRouteId, string>;
```

Comment:
this seems like a departure from the ChainFiles?

Reply:
How so?

Reply:
those are just filepaths iiuc

Reply:
They're a map of file type to file URI. These are similar, except it's ID to URI. I'll add a comment above `chains` to clarify